internal/photoprism/import_worker.go

```diff
@@ -186,11 +186,11 @@ func ImportWorker(jobs <-chan ImportJob) {
 					continue
 				}
 
-				if done[f.FileName()] {
+				if done[f.FileName()] != nil {
```

Comment:
```suggestion
				if _, ok := done[f.FileName()], ok {
```
Checks if the key is set (there is supposed to be no value)